mailpoet/tests/integration/Newsletter/AutomatedLatestContentAPITest.php

```diff
@@ -0,0 +1,228 @@
+use MailPoet\Logging\LoggerFactory;
+use MailPoet\Newsletter\AutomatedLatestContent;
+use MailPoet\Util\APIPermissionHelper;
+use MailPoet\WP\Functions as WPFunctions;
+
+class AutomatedLatestContentAPITest extends \MailPoetTest {
+
+  /*** @var array WP_User[] */
+  private $createdUsers = [];
+
+  /*** @var ALCAPI */
+  private $alcAPI;
+
+  /*** @var WPFunctions */
+  private $wp;
+
+  private static $alcBlock = [
+    'type' => 'automatedLatestContentLayout',
+    'withLayout' => true,
+    'amount' => '2',
+    'contentType' => 'post',
+    'terms' => [],
+    'inclusionType' => 'include',
+    'displayType' => 'excerpt',
+    'titleFormat' => 'h2',
+    'titleAlignment' => 'left',
+    'titleIsLink' => false,
+    'imageFullWidth' => true,
+    'titlePosition' => 'abovePost',
+    'featuredImagePosition' => 'left',
+    'fullPostFeaturedImagePosition' => 'none',
+    'showAuthor' => 'no',
+    'authorPrecededBy' => 'Author:',
+    'showCategories' => 'no',
+    'categoriesPrecededBy' => 'Categories:',
+    'readMoreType' => 'button',
+    'readMoreText' => 'Read more',
+    'readMoreButton' => [
+      'type' => 'button',
+      'text' => 'Read more',
+      'url' => '[postLink]',
+      'styles' => [
+        'block' => [
+          'backgroundColor' => '#e2973f',
+          'borderColor' => '#e2973f',
+          'borderWidth' => '0px',
+          'borderRadius' => '5px',
+          'borderStyle' => 'solid',
+          'width' => '110px',
+          'lineHeight' => '40px',
+          'fontColor' => '#ffffff',
+          'fontFamily' => 'Arial',
+          'fontSize' => '14px',
+          'fontWeight' => 'bold',
+          'textAlign' => 'left',
+        ],
+      ],
+      'context' => 'automatedLatestContentLayout.readMoreButton',
+    ],
+    'sortBy' => 'newest',
+    'showDivider' => false,
+    'divider' => [
+      'type' => 'divider',
+      'styles' => [
+        'block' => [
+          'backgroundColor' => 'transparent',
+          'padding' => '13px',
+          'borderStyle' => 'solid',
+          'borderWidth' => '3px',
+          'borderColor' => '#aaaaaa',
+        ],
+      ],
+      'context' => 'automatedLatestContentLayout.divider',
+    ],
+    'backgroundColor' => '#ffffff',
+    'backgroundColorAlternate' => '#eeeeee',
+  ];
+
+  public function _before() {
+    parent::_before();
+    $this->wp = $this->diContainer->get(WPFunctions::class);
+    $loggerFactory = $this->diContainer->get(LoggerFactory::class);
+
+    $alc = Stub::make($this->diContainer->get(AutomatedLatestContent::class), [
+      "loggerFactory" => $loggerFactory,
+      "wp" => $this->wp,
+      "transformPosts" => function ($block, $post) {
+        return $post;
+      },
+    ]);
+    $apiPermissionHelper = $this->diContainer->get(APIPermissionHelper::class);
+    $this->alcAPI = new ALCAPI($alc, $apiPermissionHelper, $this->wp);
+
+  }
+
+  public function tearDown(): void {
+    parent::tearDown();
```

Comment:
Is there a difference from using `_after`? I don't remember now...

Reply:
tearDown is after and `_before` is well before, na?

Reply:
I'm not sure I understand your answer but my question could be reformulated as "should we maybe use `_after` here?".